app/team.go

```diff
@@ -136,12 +136,12 @@ func (a *App) UpdateTeam(team *model.Team) (*model.Team, *model.AppError) {
 }
 
 func (a *App) updateTeamUnsanitized(team *model.Team) (*model.Team, *model.AppError) {
-	result := <-a.Srv.Store.Team().Update(team)
-	if result.Err != nil {
-		return nil, result.Err
+	team, err := a.Srv.Store.Team().Update(team)
```

Comment:
Probably here you can directly use `return a.Srv.Store.Team().Update(team)`

Reply:
i'm not sure, how i missed it.